packages/fxa-content-server/app/scripts/views/would_you_like_to_sync.js

```diff
@@ -40,6 +41,11 @@ const View = FormView.extend(
     submit() {
       return Promise.resolve().then(() => {
         const account = this.getAccount();
+        if (this.skipCWTS) {
+          // don't ask to specify data choices via CWTS
+          // see https://github.com/mozilla/fxa/issues/3083 for details
+          return this.onSubmitComplete(account);
+        }
 
         // we replace the current view to avoid various problems with the back button
         return this.replaceCurrentPage('choose_what_to_sync', {
```

Comment:
In other words, I don't think we ever want to go from the WYLTS screen to the CWTS screen, so I don't understand why we need a branch here that forwards to CWTS.

Reply:
good point! updated!